packages/muze-axis/src/color-axis/color-axis.js

```diff
@@ -128,12 +128,13 @@ export default class ColorAxis {
             const scale = this.scale();
             const range = scale.range ? scale.range() : null;
             const color = this._colorStrategy.value(range)(domainVal, scale, this.domain(), this.uniqueValues());
-
-            if (typeof color === 'string') {
-                const col = color.substring(color.indexOf('(') + 1, color.lastIndexOf(')')).split(/,\s*/);
-                return rgbToHsv(...col);
+            if (color) {
+                if (typeof color === 'string') {
```

Comment:
condition can merged in single if

Reply:
No, in this case it's not possible as there are 3 conditions color can be undefined or be a string or array

Reply:
in that case it is advisable that the method call  
`this._colorStrategy.value(range)(domainVal, scale, this.domain(), this.uniqueValues());`

returns a result of single type.

Reply:
Ok, this issue will be handled in #118